src/nettests/web-connectivity/package.json

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
You can probably tweak the template substitution to avoid adding an empty line here